Makefile

```diff
@@ -147,7 +147,7 @@ lint-dockerfile:
 .PHONY: fmt
 fmt:
 	@echo gofmt		# Show progress, real gofmt line is too long
-	find version test internal controllers apis -name '*.go' | xargs gofmt -s -l -w
+	find . version test internal controllers apis -name '*.go' | xargs gofmt -s -l -w
```

Comment:
nit: if we have `.` why need other folder names?

Reply:
i think we only have folders to begin with because of maybe when we had the vendor dir?